src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -454,12 +425,16 @@ function IOURequestStepConfirmation({
      * @param {String} paymentMethodType
      */
     const sendMoney = useCallback(
-        (paymentMethodType) => {
-            const currency = transaction.currency;
+        (paymentMethodType: ValueOf<typeof CONST.IOU.PAYMENT_TYPE>) => {
```

Comment:
I saw `ValueOf<typeof CONST.IOU.PAYMENT_TYPE>` several times in this PR, maybe we can create a separate type for it?

Reply:
there's only one `ValueOf<typeof CONST.IOU.PAYMENT_TYPE>`, the second usage is `DeepValueOf`